pkg/nuctl/command/import.go

```diff
@@ -249,17 +243,17 @@ Use --help for more information`)
 				return errors.Wrap(err, "Failed to identify the input format")
 			}
 
-			projectImportConfigs, err := commandeer.resolveProjectImportConfigs(projectBody, unmarshalFunc)
+			importProjectsOptions, err := commandeer.resolveImportProjectsOptions(projectBody, unmarshalFunc)
```

Comment:
Commented before but it's repeating everywhere and confusing:
importProjectX -> projectImportX
in all var and types everywhere